.github/workflows/run-weekly-rv64gcv-zvl-lmul2.yaml

```diff
@@ -13,3 +13,5 @@ on:
 jobs:
   init-submodules:
     uses: ./.github/workflows/init-submodules.yaml
+    with:
+      prefix: 'rv64_zvl_lmul2_'
```

Comment:
[just a comment, not a change request] whoops, didn't realize this was an unimplemented weekly run - probably cause there wasn't a way of passing in testsuite command line args back then